### PR TITLE
Navigate on QR display scan

### PR DESCRIPTION
**Pull request by ryankoch13**
# Summary of Changes

App will go to home screen if the invite it displays is scanned and connected to

# Related Issues

[HOL-45](https://indicio.atlassian.net/browse/HOL-45?atlOrigin=eyJpIjoiZmUwYmMxZmRiMzE3NGE5Yjk0NGFhOWE5YzEwNjhmMmYiLCJwIjoiaiJ9)

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [x] Run prettier: `npm run style-format`
- [x] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/screens/Chat.tsx
Comment (JamesKEbert) [inline core/App/screens/Chat.tsx:42]: Does this impact the out of order chat messages? Was this intentional?

**Review by JamesKEbert — COMMENTED** [review]
Code looks good (besides one comment for discussion), needs testing

File: core/App/screens/Chat.tsx
Comment (ryankoch13) [reply core/App/screens/Chat.tsx:42]: I was hoping it would impact the out of order messages, but I wasn't able to replicate so I couldn't verify. it doesn't seem to hurt anything at least

**Review by sabejensen — APPROVED** [review]
Tested, LGTM~ can merge once package-lock merge conflict is resolved

**Pull request merged**
